specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/Jobs.json

```diff
@@ -480,21 +448,7 @@
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
           },
           {
-            "name": "jobName",
-            "in": "path",
-            "description": "Name of the Container Apps Job.",
-            "required": true,
-            "type": "string",
-            "pattern": "^[-\\w\\._\\(\\)]+$"
-          },
-          {
-            "name": "jobExecutionName",
-            "in": "body",
-            "description": "List of all job executions that should be stopped.",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/JobExecutionNamesCollection"
-            }
+            "$ref": "#/parameters/JobNameParameter"
```

Comment:
Removing body parameter is a breaking change.

Reply:
I am not removing the job name path parameter. We use this parameter in a few places and it has been copied and pasted at each location. So I moved it to the parameters section and put a "#/parameters/JobNameParameter" in each place it was previously used.